packages/eslint-config/index.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
       rules: {
         'no-undef': 'off',
         '@redwoodjs/redwood/no-unavailable-pages': 'error',
+        'jsx-a11y/aria-role': 'off',
```

Comment:
How about `ignoreNonDOM`?

Reply:
Also an option; would cover the "arbitrary code" case—your thoughts as well?

Reply:
Yeah, exactly.

Still not perfect. And some people like to argue "don't do it if you can't do it right". Whereas others like to arge "better than nothing". Your call.

Reply:
Considering anything goes in the Router it seems like the better option!